src/modules/aws_security_group@0.0.1/entity/index.ts

```diff
@@ -12,19 +13,14 @@ import { cloudId, } from '../../../services/cloud-id'
 
 @Entity()
 export class SecurityGroup {
-  @PrimaryGeneratedColumn()
-  id?: number;
+  @PrimaryColumn()
+  groupName?: string;
```

Comment:
Should not be nullable anymore, right?
```suggestion
  groupName: string;
```